parts/22-query.sh

```diff
@@ -952,6 +952,47 @@ query_tool-last-used-date() { ## : When was the most recent invocation of every
 	EOF
 }
 
+local_query-toolusebygroup() { ## : <year-month> <group> Lists count of tools used by all users in a group
```

Comment:
```suggestion
query_tool-use-by-group() { ## : <year-month> <group> Lists count of tools used by all users in a group
```

local is only used for local commands, query must be followed by a `_`, and then generally we use `-` to separate words in a tool command

Reply:
```suggestion
local_query-toolusebygroup() { ##? <year_month> <group>: Lists count of tools used by all users in a group
```
The : needs to come at the end. Additionally I'm making a suggestion to use `##?` which invokes our automatic command line parsing, saving you the bother of doing error checking.

Reply:
Hi hexylena.  As per your suggestion, this is now fixed.